.changeset/small-ties-sort.md

```diff
@@ -0,0 +1,50 @@
+---
+'astro': major
+---
+
+Fixes attribute rendering for non-[boolean HTML attributes](https://developer.mozilla.org/en-US/docs/Glossary/Boolean/HTML) with boolean values to match proper attribute handling in browsers.
+
+Previously, non-boolean attributes may not have included their values when rendered to HTML. In Astro v5.0, the values are now explicitly rendered as `="true"` or `="false"`
+
+In the following `.astro` examples passed values of `true` and `false`, only `allowfullscreen` is a boolean attribute:
```

Comment:
```suggestion
In the following `.astro` examples, only `allowfullscreen` is a boolean attribute:
```
I know it was my line, but I kept stumbling over it every time I read it. It's probably good enough just to leave it out!